README.md

```diff
@@ -245,7 +245,7 @@ line in your terminal window, we give them on multiple lines to ensure that
 they are easily viewed on a wide variety of displays.
 
 ```
-pipenv run python3 gatorgrader.py \
+$ pipenv run python3 gatorgrader.py \
```

Comment:
This looks good. Placing them on each line would make copy-pasting inconvenient, so I think this strategy should work well!

Reply:
Great, thanks for the confirmation.